### DISCUSSION TITLE
Messaging policy

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#77] [Qualified]
### What are some documents containing these terms?

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | Twilio | [URL 🔗](https://www.twilio.com/en-us/legal/messaging-policy) | Messaging Policy | 🇺🇸 |
| 2 | OnStar | [URL 🔗](https://www.onstar.com/legal/onstar-text-messaging-terms-and-conditions) | Texting Policy | 🇺🇸 🇨🇦 |
| 3 | Aspiration | [URL 🔗](https://assets.ctfassets.net/ejg6grzpbbxj/7s85knyAH3EfrYRiVfo06Z/36d8939151e18dbbbe38c520c224f190/Aspiration_Financial__LLC___SMS_Policy._08.06.21.pdf) | SMS policy | 🇺🇸 |
| 4 | KrispCall | [URL 🔗](https://krispcall.com/legal/messaging-policy/) | Messaging Policy | 🌐 |
| 5 | RingCentral | [URL 🔗](https://www.ringcentral.com/legal/sms-mms-content-policies.html) | RingCentral SMS/ MMS content policies | 🌐 |

### What could be some alternative names for this type?

SMS policy
Texting policy

### Whose rights and duties are defined in these terms?

end user

### What is the object of commitment in these terms?

managing text messaging

**Comment by Ndpnt** [comment]
One example is missing to move this discussion as `Qualified`

**Reply by clementbiron** [reply]
I've added one.

**Comment by MattiSG** [comment]
- Twilio, KrispCall and RingCentral are messaging providers. Their “messaging policy” thus seems to me to be just a part of their Acceptable Usage Policy.
- OnStar considers its “[Texting User Terms](https://www.onstar.com/us/en/onstar-text-messaging-terms-and-conditions.html)” as “[Additional User Terms](https://www.onstar.com/legal/user-terms)”, and I thus see them as a multi-document ToS.
- Aspiration’s policy is just a short one with points that relate to Privacy Policy and others that relate to ToS. In either case, I see them more as part of a multi-document ToS or PP.

The current suggested object of commitment outlines (“managing text messaging”) sounds too broad to me.

I don't see this suggestion as relevant, at least as it currently is, as it puts together very different commitments, just because the same underlying technology is used (text messages).
